bridge_test.go

```diff
@@ -51,6 +53,47 @@ func TestHandlePacket(t *testing.T) {
 				&TimerEvent{
 					metricName: "foo",
 					value:      200,
+					labels:     map[string]string{},
+				},
+			},
+		}, {
+			name: "datadog tag extension",
+			in:   "foo:100|c|#tag1:bar,tag2:baz,tag3,tag4",
+			out: Events{
+				&CounterEvent{
+					metricName: "foo",
+					value:      100,
+					labels:     map[string]string{"tag1": "bar", "tag2": "baz", "tag3": ".", "tag4": "."},
+				},
+			},
+		}, {
+			name: "datadog tag extension with # in all keys (as sent by datadog php client)",
+			in:   "foo:100|c|#tag1:bar,#tag2:baz,#tag3,#tag4",
+			out: Events{
+				&CounterEvent{
+					metricName: "foo",
+					value:      100,
+					labels:     map[string]string{"tag1": "bar", "tag2": "baz", "tag3": ".", "tag4": "."},
```

Comment:
I'm thinking whether `.` makes it clear that this is a key only label. What about something like `<n/a>`?

Also, is it common that implementation use many different varying tags here? This would be problematic, as each unique label set will result in a new time series. It's also not allowed to use metrics with varying label sets in general.